napari_allencell_annotator/_tests/widgets/annotation_widget_test.py

```diff
@@ -1,9 +1,63 @@
 from unittest import mock
 from unittest.mock import MagicMock, create_autospec
 
+from napari_allencell_annotator.model.combo_key import ComboKey
+
+from napari_allencell_annotator.model.key import Key
 from qtpy.QtCore import QSize
 
 from napari_allencell_annotator.widgets.annotation_widget import AnnotationWidget, AnnotationItem
+import pytest
+
+
+@pytest.fixture
+def annotation_widget(qtbot) -> AnnotationWidget:
+    return AnnotationWidget()
+
+
+def test_add_existing_item_string(annotation_widget: AnnotationWidget) -> None:
+    # ACT
+    annotation_widget.add_existing_item("test_string", Key("string", "test_value"))
+
+    # ASSERT
+    assert annotation_widget.count() == 1
+    assert annotation_widget.item(0).name.text() == "test_string"
```

Comment:
dont need to assert item state imo

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/92/commits/c7d87a520c34ddc25c82b0943d637bc2c3985e6d